ArcGISRuntimeSDKQt_CppSamples/Layers/DisplaySubtypeFeatureLayer/DisplaySubtypeFeatureLayer.cpp

```diff
@@ -35,13 +35,14 @@ using namespace Esri::ArcGISRuntime;
 DisplaySubtypeFeatureLayer::DisplaySubtypeFeatureLayer(QObject* parent /* = nullptr */):
   QObject(parent),
   m_map(new Map(Basemap::streetsNightVector(this), this)),
+  m_cred(new Credential("viewer01", "I68VGU^nMurF", this)),
```

Comment:
We should append to the readme about the credentials required in an additional information section, just as we do for Edit with Branch Versioning sample

Reply:
This should be done for all samples that didn't used to need credentials but now do - I didn't mark them all explicitly so as not to be overly verbose, but please address throughout